### DISCUSSION TITLE
Played list should show most recently played first

### DISCUSSION THREAD

**Discussion by lordbah** [music-assistant/support#3189] [Feature requests and ideas]
Played list should show most recently played track at the top. I believe the most frequent use case for Played is someone wants to see what track was just played. If he has been playing a playlist for a couple of hours, he doesn't want to have to scroll to the bottom of that list to see what was just played, which is currently the case.

**Comment by OzGav** [comment]
Note sure where you are looking. The tracks list can be sorted by last played descending or the Home Screen shows the most recent?

**Comment by lordbah** [comment]
Here. If I want to know what song just played, I have to scroll that Played list all the way down to the end.
![Screenshot from 2025-02-22 12-12-43](https://github.com/user-attachments/assets/155b7603-c929-43de-8e60-7ab4c7c23790)

**Reply by marcelveldt** [reply]
This makes a lot of sense to me.
I have just implemented this - simply sort the "played" list in reverse so the last played item is at the top.
If this may lead to complaints of people who actually want it the other way around, I'll make this a toggle.

Available in next release (2.4.3 stable or 2.5.0b3 beta)

**Comment by OzGav** [comment]
The feature request has been implemented successfully. As such this feature request will be closed and locked. If there are enhancements desired then open a new [feature request](https://github.com/orgs/music-assistant/discussions/categories/feature-requests-and-ideas). If there is a question about how this works then check the [documentation](https://beta.music-assistant.io/), or ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord. If there are problems then ask a question in the [Q&A](https://github.com/orgs/music-assistant/discussions/categories/q-a) area or on Discord or open an [issue](https://github.com/music-assistant/support/issues). Thanks for your interest in seeing this idea being brought to life.

Note since the original idea the UI has changed to one continuous list.
